src/sentry/backup/imports.py

```diff
@@ -441,8 +441,33 @@ def import_in_global_scope(
     Sentry instance, some behaviors in this scope are different from the others. In particular,
     superuser privileges are not sanitized. This method can be thought of as a "pure"
     backup/restore, simply serializing and deserializing a (partial) snapshot of the database state.
+    During the setup of a fresh Sentry instance, there are a couple of models that are automatically created.
+    The Sentry org, a Sentry team, and an internal project. During a global import, we want to avoid persisting
+    these default models and start from scratch.
     """
 
+    from django.core.management.color import no_style
+    from django.db import connection
+
+    from sentry.models.organization import Organization
+    from sentry.models.organizationmember import OrganizationMember
+    from sentry.models.project import Project
+    from sentry.models.projectkey import ProjectKey
+    from sentry.models.team import Team
+
+    models_to_delete = [Project, ProjectKey, Organization, OrganizationMember, Team]
+
+    # Global imports will never be run in production, ever
+    with unguarded_write(using="default"):
+        for model in models_to_delete:
+            model.objects.all().delete()
```

Comment:
This won't be sufficient for all models, since some of them use `Paranoid` managers instead (see the above referenced `clear_database` function for more on this).